apps/token/src/routes/governance/proposal/proposal-container.tsx

```diff
@@ -0,0 +1,81 @@
+import { gql, useQuery } from '@apollo/client';
+import { Callout, Intent, Splash } from '@vegaprotocol/ui-toolkit';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useParams } from 'react-router-dom';
+
+import { SplashLoader } from '../../../components/splash-loader';
+import useFetch from '../../../hooks/use-fetch';
+import { getDataNodeUrl } from '../../../lib/get-data-node-url';
+import { Proposal } from '../components/proposal';
+import { PROPOSALS_FRAGMENT } from '../proposal-fragment';
+import type {
+  Proposal as ProposalQueryResult,
+  ProposalVariables,
+} from './__generated__/Proposal';
+
+/**
+ * TODO: how do we do this properly to ensure that it is kept up to date?
```

Comment:
can we add a ticket for this, as generate REST data types from swagger

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/271